new/prusti-specs/src/specifications/preparser.rs

```diff
@@ -0,0 +1,605 @@
+/// The preparser splits a composite Prusti assertion into atomic assertions, parses the resulting
+/// Rust expressions, and then assembles the composite Prusti assertion.
```

Comment:
Could you explain the relation between *atomic assertion* and *Rust expression*.